### DISCUSSION TITLE
Able to remove grey comments like refresh emotes comments

### DISCUSSION THREAD

**Discussion by 158flairincident** [Chatterino/chatterino2#3670] [Ideas] [answered]
![chat2](https://user-images.githubusercontent.com/103391465/162746557-1e9e42a7-0b90-44d3-93e6-13eafaec2a98.png)

**Comment by Felanbird** [comment, marked as answer]
`!flags.system_message` add this filter

**Reply by 158flairincident** [reply]
where do you go from here?  

![chat3](https://user-images.githubusercontent.com/103391465/163680981-1b8dd6ea-9d67-4ba8-9bfe-cafccde1124b.png)

**Reply by KararTY** [reply]
> 

Use "Quick Add" button in the top right of Filters.
And then just enter that in the middle input bar.
![image](https://user-images.githubusercontent.com/13697809/163681698-61c1ab18-b648-4e4a-b60a-a869c2867c57.png)
